changelog.d/6149.bugfix

```diff
@@ -0,0 +1 @@
+Make widget web view request system permissions for camera and microphone (PSF-1061)
```

Comment:
Happy to remove it but having it in the changelog / release notes would allow us to directly cross reference with Jira. I think it's essentially also not better or worse than including it in commits and pull requests – which we already do today.

Reply:
This will be visible from the release page, so would be better to have a corresponding github issue...

Reply:
I assumed with the filename it'll link to this PR and, therefore, had intentionally explained the issue that is fixed in detail up in the description to avoid having to create a separate issue just to close it again.